.github/workflows/submit_firefox.yml

```diff
@@ -42,5 +42,7 @@ jobs:
             approval-note: |
               The source code and build instructions are available at 
               https://github.com/csfloat/extension?tab=readme-ov-file#how-to-build-release.
+            release-note: |
+              Learn more about this release and its changes from the latest [Github release](https://github.com/csfloat/extension/releases).
```

Comment:
I think we should reference the version directly since otherwise it'll look a bit odd to see "latest" for every version on https://addons.mozilla.org/en-CA/firefox/addon/csgofloat/versions/

Looks like `robinraju/release-downloader` exposes `tag_name` and `release_name` (https://github.com/robinraju/release-downloader?tab=readme-ov-file#output-variables).

So we could do something like:

```
Learn more about release ${{ steps.release.outputs.release_name }} by visiting the [GitHub Release](https://github.com/csfloat/extension/releases/tag/${{ steps.release.outputs.tag_name }})
```